docs/setup.md

```diff
@@ -48,7 +48,7 @@ $ echo 'eval "$(direnv hook bash)"' >> ~/.bashrc
 
 ```bash
 $ cd ~/workspace/calltocode.org
-$ nvm install
+$ npm install
```

Comment:
`nvm install` was here to install the node version specified in `.nvmrc`.

Reply:
oh, ok....cancel then the PR